src/components/events/YearlyEvents.svelte

```diff
@@ -4,8 +4,10 @@
 
 	export let events: YVEvent[];
 
-	let for_all = true;
-	let only_members = true;
+	let filter_for_all = true;
```

Comment:
I think we should talk with Emma about the wanted logic here.

I've described my understanding in #49 but that might not be what Emma had in mind. Currently there is no way to see all events at the same time though.

Reply:
I only now realize how many different ways there are how you could do it.
Here is what Emma wrote on figma:
ich dachte das es ehr so waere das man nuer entwieder nichts oder eine sache auswaehlen kann
und das default is das keins ausgewaehlt ist

Reply:
Yeah, I talked with Emma about it and she said might have to test the different ways. She also said she likes your interpretation the most right now, so perhaps you're on the best track ;)

Reply:
We'll do it like this for now, Emma said she likes it.